internal/crowdactions/crowdaction.go

```diff
@@ -0,0 +1,81 @@
+package crowdaction
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/CollActionteam/collaction_backend/internal/constants"
+	"github.com/CollActionteam/collaction_backend/internal/models"
+	"github.com/CollActionteam/collaction_backend/utils"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/aws/aws-sdk-go/service/dynamodb/expression"
+)
+
+const Separator = "### app version:"
+
+type DynamoRepository interface {
```

Comment:
The interface should only use non-aws types.  
(Probably should also be called something like `CrowdactionRepository` or `DatabaseRepository`, but I am not sure about this part of the architecture. Maybe @mrsoftware has an opinion about this.)

Reply:
Oh okay, maybe I was confused but I thought they would be the same functions as in the repository. 

For example if you look at the internal/contact/email.go, you can see that EmailRepository is declaring a Send func, which is then being used in the SendEmail function.

This Send func is the same as there is in the in the pkg/repository/aws/emailManager.go file.

Reply:
Yes. `internal/contact/...` contains the services with the interfaces for repositories (ports) and `pkg/repository/...` contains the corresponding adapters.

Reply:
Perfect then, I understand now. This was an issue on my side for misinterpreting the naming convention for the specific type. I have fixed it now.